src/data/equiv/ring.lean

```diff
@@ -241,7 +241,7 @@ lemma to_ring_hom_injective : function.injective (to_ring_hom : (R ≃+* S) →
 
 instance has_coe_to_ring_hom : has_coe (R ≃+* S) (R →+* S) := ⟨ring_equiv.to_ring_hom⟩
 
-lemma to_ring_hom_eq_coe (f : R ≃+* S) : f.to_ring_hom = ↑f := rfl
+@[simp] lemma to_ring_hom_eq_coe (f : R ≃+* S) : f.to_ring_hom = ↑f := rfl
```

Comment:
Isn't this going to make the linter unhappy? We have a lot of lemmas stated in terms of `f.to_ring_hom` that will need to change to be simp-normal.

Reply:
Yes, possibly, I don't know this area well-enough. Since the lemma was stated in this direction, I thought that the coercion was the normal form. If the linter complains, I'll just remove this.

Reply:
I think the lemmas should be stated in terms of the coercion instead of `to_ring_hom`, so making this lemma here simp is a good idea.

Reply:
There was a discussion about this [here](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/simp-normal.20form.20for.20bundled.20hom.20conversion/near/226552455). At any rate, this has nothing to do with finsets so should probably be its own PR.

Reply:
Removed from this PR.